app/helpers/application_helper.rb

```diff
@@ -527,6 +527,16 @@ def bracket_small_population(value, mask: true)
     end
   end
 
+  # page title with optional block
+  # = render_page_title "New Report" do
+  #   = render 'type_warning'
+  def render_page_title(title, &block)
```

Comment:
maybe this should be:
```suggestion
  def render_page_heading(title, &block)
```
Just to differentiate from the title tag which is what we're fixing?

Reply:
good call, how about render_display_title